src/main/java/com/rental/rentalApp/RentalAppApplication.java

```diff
@@ -1,7 +1,9 @@
 package com.rental.rentalApp;
 
 import com.rental.rentalApp.entities.Client;
+import com.rental.rentalApp.entities.Status;
 import com.rental.rentalApp.repositories.ClientRepository;
+import com.rental.rentalApp.repositories.StatusRepository;
```

Comment:
To avoid merge conflicts

```suggestion
import com.rental.rentalApp.entities.*;
import com.rental.rentalApp.repositories.*;
```